wikitemplate.md

```diff
@@ -138,12 +155,15 @@
 8. [ ] Open an email on http://mail.google.com/ or inbox.google.com and click on a link. Make sure it works.
 9. [ ] Test that PDF is loaded at http://www.orimi.com/pdf-test.pdf
 10. [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run).
+11. [ ] Test that extension icons can be displayed on the navigation bar
 
 ## Flash tests
+
 1. [ ] Turn on Flash in about:preferences#security. Test that clicking on 'Install Flash' banner on myspace.com shows a notification to allow Flash and that the banner disappears when 'Allow' is clicked.
```

Comment:
This is also obselete. It now shows Cookie banner on the site. Can this be removed?

Reply:
Let's open an issue for that :-)